source/retryable-reads/tests/aggregate-merge.yml

```diff
@@ -0,0 +1,39 @@
+runOn:
+  -
+    minServerVersion: "4.1.12"
```

Comment:
FWIW, neither `$out` nor `$merge` can create a new output collection on a sharded cluster. That said, these tests expect an error and the fail point aborts the `aggregate` command so I don't think it's a concern.

Reply:
Your comment suggests that this test will pass with and without the configureFailPoint. Can you could add an insert to create the output collection before running the `$merge`? That would avoid conflating the closeConnection error and the $merge error.

Reply:
I think my previous comment was mistaken. [`$out`](https://docs.mongodb.com/v4.0/reference/operator/aggregation/out/) cannot output to a _sharded_ collection, but it _can_ create a new collection on a sharded cluster. Since there is no reason for the namespaces in these `$out` and `$merge` tests to be configured as a sharded collection, I don't think there's a problem here. Moreover, it looks like [`$merge`](https://docs.mongodb.com/master/reference/operator/aggregation/merge/) actually supports outputting to a sharded collection, so this concern certainly shouldn't apply to _this_ test.

The JSON for this `$merge` test is identical to the corresponding `$out` test in `aggregate.json` apart from the pipeline stage, description, and order of `name` and `object` fields.

Reply:
Oh great!